plugins/content_types/search_result.inc

```diff
@@ -83,10 +83,22 @@ function ting_search_search_result_content_type_render($subtype, $conf, $panel_a
 
     // Set message.
     $message = '';
+
+    $frequency_rank = array(
+      'rank_main_title' => t('Title'),
+      'rank_creator' => t('Creator'),
+      'rank_subject' => t('Subject')
+    );
+    if (isset($search_result) && in_array($search_result->sortUsed, array_keys($frequency_rank))) {
+      $msg = t('Records are sorted by %sortUsed . Select another sort if it is not suited', array('%sortUsed' => $frequency_rank[$search_result->sortUsed]));
+      $message .= '<div class="messages status">' . $msg . '</div>';
```

Comment:
You are right. I did not spot the fact that is is separate strings being concatenated.